app/main.py

```diff
@@ -2,5 +2,15 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_results = dict()
+
+    def inner(*args) -> int:
```

Comment:
You can receive here any type of data ) Add pls type hint on 'args' , and change the 'int' type hint